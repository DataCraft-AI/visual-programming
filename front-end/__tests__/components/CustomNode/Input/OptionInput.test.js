import React from 'react'
import { shallow, mount } from 'enzyme';
import OptionInput from '../../../../src/components/CustomNode/Input/OptionInput';

describe('Validates OptionInput', () => {
  it('Display OptionInput', () => {
    const optionInput = shallow(<OptionInput isFlow={false} />);
    expect(optionInput).toMatchSnapshot();
  });
});
