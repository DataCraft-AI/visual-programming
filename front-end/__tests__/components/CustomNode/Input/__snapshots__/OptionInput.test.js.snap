// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Validates OptionInput Display OptionInput 1`] = `ShallowWrapper {}`;
